### ISSUE TITLE
Duplicate images in a presentation only show 1 time.

### ISSUE THREAD

**Issue by robb-price** [Rise-Vision/rise-image#109]
### What is the expected behavior?
Regardless if a picture is in a presentation several times, it should show each time it comes up in the playlist.

### What is the actual behavior?
If you have an image in a playlist more than 1 time, it will only show the first time, and never after it.

### What steps can be taken to replicate the problem?
Use this presentation to test: https://apps.risevision.com/templates/edit/2cf9e40e-a881-4762-a438-332dbb76d292/?cid=556323b8-984c-4127-aa08-4d73d019bcb8

**Comment by stulees** [comment]
@robb-price What is the priority of this defect? Can it be deferred to our next Cooldown and closed by 24-June?

**Comment by robb-price** [comment]
@stulees I believe that will work for the user, but I will confirm with them and let you know. Thanks!

**Comment by robb-price** [comment]
@stulees The user confirmed they are cool with waiting on this one. So it can be deferred and closed by the 24th of June.

**Comment by stulees** [comment]
Thanks @robb-price

**Comment by stulees** [comment]
@robb-price Closed with #112
